database/seed.js

```diff
@@ -18,33 +19,65 @@ const generateRecords = function(numRecords) {
     records.push({
       id: i,
       title: faker.commerce.productName(),
-      brand: brands[ Math.floor( Math.random() * brands.length ) ],
-      department: departments[ Math.floor( Math.random() * departments.length ) ],
+      brand: brands[Math.floor(Math.random() * brands.length)],
+      department: departments[Math.floor(Math.random() * departments.length)],
       price: Number(faker.commerce.price(0, 100)) - Math.ceil(Math.random() * 5) / 100,
       imageUrl: `https://twzkraus-fec-images.s3-us-west-1.amazonaws.com/target-images/${i % 50}.jpg`,
       productUrl: `/${i % 100 + 1}`
     });
   }
+  console.log(records.length)
   return records;
 };
 
-RecommendedItem.deleteMany({}, (err) => {
-  if (err) {
-    console.log('error deleting:', err);
-  } else {
-    console.log('database has been cleared and will be re-generated');
-    const records = generateRecords(700);
-    RecommendedItem.create(records)
-      .then(() => {
-        fs.writeFile('./data/example_data.js', `module.exports = ${JSON.stringify(records)}`, (err) => {
-          if (err) {
-            console.log('error:', err);
-          } else {
-            console.log('example data file has been written');
-          }
-          mongoose.disconnect();
-        });
-      });
+const seedController = (args) => {
+  let numOfRecords = args[0] || 700;
+  let chunkSize = args[1] || numOfRecords;
+  let callback = callBackGenerator(args[2]);
+  let chunks = numOfRecords / chunkSize;
```

Comment:
could chunks end up being a fraction here? maybe that wouldn't matter anyway.

Reply:
Good Catch! I actually forgot to remove this file from my commit as it is going through a heavy revision now as well...